Added an example to test the line-to-line-angle Jacobian and its resi…

juanjqo opened the pull request:
@dqrobotics/developers 

Hi @mmmarinho,

I added the python version of the example recently added in C++, which is used to test the line-to-line-angle Jacobian and its residual.

Best regard, 

Juancho

juanjqo committed:
Added an example to test the line-to-line-angle Jacobian and its residual.

mmmarinho commented on vrep_interface_tests/line_to_line_angle_jacobian/line_to_line_angle_jacobian.py:
```diff
@@ -0,0 +1,166 @@
+    safe_angle = 15*(pi/180)
+    T = 0.005
+    w = 0.2
+    Alpha = 20*(pi/180)
```
The uppercase `Alpha` is weird. Any special reason?

juanjqo replied:
@mmmarinho 
There is no special reason. I guess I was using the classic style definition of a sinusoid in which the amplitude is represented with an uppercase letter.

Source:  [Matlab documentation](https://www.mathworks.com/help/dsp/ref/dsp.sinewave-system-object.html)
![Screenshot 2022-11-17 105959](https://user-images.githubusercontent.com/23158313/202337890-9546b167-098c-46bb-84a1-b4ff91f23312.png)

However, to keep the convention of uppercase for matrices and lowercase for scalars, I modified the variable from `Alpha` to `alpha`.

Best regards, 

Juancho

bvadorno replied:
Sorry, @juanjqo, but I have to comment on this one. The classic definition uses the Latin letter `A` for the amplitude, not `Alpha`! 🤣

If this is a local variable that refers to the amplitude, then I suggest using `amplitude`.

Best,
Bruno

juanjqo replied:
@bvadorno you're right! 😂

juanjqo committed:
[line_to_line_angle_jacobian.py] Modified the variable Alpha to alpha.

mmmarinho commented:
@juanjqo 
Let's change it to `amplitude` so that I can accept this pull request!
(Instead of the current `alpha` I believe).

juanjqo committed:
Updated the example according to suggestions by Murilo and based on to the new version of the cpp example.

juanjqo commented:
@mmmarinho I implemented your suggestion and updated the code based on the current version of the example in C++.
I tested the example again and it is working.

Best regards, 

Juancho

juanjqo committed:
Removed unused import
